lib/active_encode/status.rb

```diff
@@ -7,17 +7,13 @@ module Status
     included do
       # Current state of the encoding process
       attr_accessor :state
-      attr_accessor :current_operations
-      attr_accessor :percent_complete
       attr_accessor :errors
 
       attr_accessor :created_at
-      attr_accessor :finished_at
       attr_accessor :updated_at
-    end
 
-    def created?
-      !id.nil?
+      # @deprecated
+      attr_accessor :finished_at
     end
 
```

Comment:
do we want to provide a getter for finished_at which returns updated_at if state is complete, or  nil otherwise?

Reply:
We could but since that attribute is getting deprecated I'm more inclined to just leave it as it has been.